README.md

```diff
@@ -8,14 +8,19 @@ Most importantly, `cloudwash` provides the CLI utility called '`swach`' that can
 
 cloudwash supports following cloud providers:
 
-* Amazon EC2
-* Google Cloud
-* Microsoft Azure
+| Cloud Providers  | vms    | nics    | discs     | images   | pips    | stacks   |
+|------------------|--------|---------|-----------|----------|---------|----------|
+| Amazon EC2       | &check;| &check; | &check;   |  &check; |  &check;|  &check; |
+| Google Cloud     | &check;| &check; | &check;   |  &cross; |  &cross;|  &cross; |
+| Microsoft Azure  | &check;| &check; | &check;   |  &check; |  &check;|  &cross; |
+
```

Comment:
Add new Compute Resource `VMWare` here as we have recently added the support there.